deployment-examples/gcp/k8s/deploy_to_gke.sh

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+
+set -e
+
+## edit those environment variables
+## ----------------------
+export GCLOUD_PROJECT_ID="YOUR_PROJECT_ID"
+export GCLOUD_ZONE="europe-west3-c"
+export GCLOUD_CLUSTER_NAME="commercetools-subscriptions"
+export IMAGE_BASE_NAME="commercetools-subscriptions"
+export TAG="v0.0.1"
+export GCR_PATH="eu.gcr.io"
+## ----------------------
+
+export HELM_HOME="$HOME/helm"
+export HELM_VERSION="v3.5.2"
+export HELM_VALUES_DIR="./k8s"
+export HELM_CHARTS_REPO="https://github.com/commercetools/k8s-charts.git"
+export HELM_CHARTS_VERSION="1.14.0"
+export HELM_CHART_TEMPLATE_NAME="cronjob"
+export HELM_CHARTS_LOCAL_FOLDER="charts-templates"
+
+printf "\n- Login to gcloud SDK and select project \n"
+gcloud config set project "$GCLOUD_PROJECT_ID"
+gcloud config set compute/zone "$GCLOUD_ZONE"
+
+printf "\n- Build and push docker images to Google Container Registry(eu.gcr.io) \n"
+IMAGE_FULL_NAME="${GCR_PATH}/${GCLOUD_PROJECT_ID}/${IMAGE_BASE_NAME}"
+
+docker build -t "$IMAGE_BASE_NAME" ./../../../
```

Comment:
Actually I would use the image from the open-source repo that @praveenkumarct was talking about before, so that you don't have to build the image every time.

Reply:
I [created a repository](https://hub.docker.com/r/commercetools/commercetools-subscriptions) on docker-hub, made it a process for [github action](https://github.com/commercetools/commercetools-subscriptions/blob/add-deployment-example-for-gcp-k8s/.github/workflows/docker-push.yml) (triggers on new tag) also [documented ](https://github.com/commercetools/commercetools-subscriptions/pull/22/commits/14e97b67dce7bebebd0c95b1e51d35b218e0c793)it.

For this comment, it could stay as is, building something every time is not issue, it's just an example script, and also shows how you might use your registry on GCP, so will not update this part.